mouse-look.ts

```diff
@@ -38,71 +40,115 @@ export class MouseLookComponent extends Component {
     private rotationX = 0;
     private rotationY = 0;
     private mouseDown = false;
+    /** Pointerlock spec prevents calling pointerlock right after user exiting it via esc for ~1 second */
```

Comment:
Do you have a link for that? Better would be to know the exact moment a next pointer lock is possible. I would bet it's after flushing the task queue, rather than a delay?

Reply:
@DavidPeicho I learned about it in a quora discussion , was not able to find its official documentation.

Reply:
@DavidPeicho  https://discourse.threejs.org/t/how-to-avoid-pointerlockcontrols-error/33017/2#:~:text=Sorry%2C%20no%20way%20to%20avoid%20it.%20As%20per%20the%20spec%3A